Snowdog_Components/components/Organisms/footer/footer.hbs

```diff
@@ -1,23 +1,31 @@
 <footer class="footer {{ class }}" {{{ attributes }}}>
     {{#if handler }}
         <div class="footer__handler {{ handlerClass }}">
-            {{ render '@button--icon' scrollToTop }}
+            {{#if scrollToTop}}
+                {{ render '@button--icon' scrollToTop }}
+            {{/if}}
 
-            <section class="footer__newsletter">
-                {{ render '@newsletter' }}
-            </section>
+            {{#if newsletter}}
+                <section class="footer__newsletter">
+                    {{ render '@newsletter' }}
+                </section>
+            {{/if}}
 
             <div class="footer__social-handler {{ socialHandlerClass }}">
                 <h3 class="footer__social-list-title">
                     {{ socialListTitle }}
                 </h3>
 
-                {{ render '@list--icon' socialIconList }}
+                {{#if socialIconList}}
```

Comment:
I think that the whole `footer__social-handler` section shouldn't be displayed if `socialIconList` is empty